src/components/Header/Header.css

```diff
@@ -1,41 +1,16 @@
-.Header nav a {
-    display: inline-block;
-}
-
-.Header nav a:hover{
-    background-color: #fff;
-    color:#000;
-    border-radius: 0;
-    cursor: pointer;
-}
-
-.Header .menuButton {
-    display: none;
-}
-
-.Header .inputButton {
-    display: none;
+.Header .user-profile {
+    &:hover .ProfileSideNav {
+        display: flex;
+    }
 }
-
 /* Mobile */
 
 @media screen and (max-width: 400px) {
 
-    .Header nav {
-        display: none
-    }
-
-    .Header .nav--open {
-        display: flex;
-        flex-direction: column;
-    }
-
     .Header .logo {
-    display: none;
+    width: 55px;
```

Comment:
Can't we do this using Tailwind CSS and putting a responsive className directly on the logo?